erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -98,24 +98,203 @@ void BandwidthEstimationHandler::updateExtensionMap(bool is_video, std::array<RT
   }
 }
 
+// Implements x < y, taking into account RTP sequence number wrap
+// The general idea is if there's a very large difference between
+// x and y, that implies that the larger one is actually "less than"
+// the smaller one.
+//
+// I picked 0x8000 as my "very large" threshold because it splits
```

Comment:
prefer setting a self-explicative constant like `uint const kSequenceNumberThreashold = 0x8000` or something like that. You can also add a comment saying that it splits 0xffff too IMO.

You can remove the rest of the comment here.